spec/features/monitor/sp_signin_spec.rb

```diff
@@ -13,7 +13,7 @@
       visit_idp_from_oidc_sp
       sign_in_and_2fa(monitor.config.login_gov_sign_in_email)
 
-      if page.has_content?('You are now signing in for the first time')
+      if on_consent_screen?
```

Comment:
rubocop doesn't like this 😭 ... ACAB

Reply:
grumble grumble grumble. This is one of my least favorite linter rules. The actual flow is backwards. The interpreter evaluates the conditional first.

Reply:
We should change the lint rules, the linter isn't in charge, us humans are!